app.js

```diff
@@ -168,11 +168,22 @@ var eventsModule = function () {
     // used to return all of the EF templates used as array
     function GetTemplates()
     {
-        return Object.keys(efDataHolder).unshift("None");
+        templates = ["None"];
```

Comment:
i'm not sure i like "None" being in the code, i think it should probably be in the UI stuff, but i'm not going to worry about it for this project.